mtp_api/apps/performance/views.py

```diff
@@ -172,13 +172,19 @@ class DigitalTakeupReport(AdminViewMixin, TemplateView):
     template_name = 'performance/digital-takeup-report.html'
     required_permissions = ['transaction.view_dashboard']
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.exclude_private_estate = True
+
     def get_context_data(self, **kwargs):
         context_data = super().get_context_data(**kwargs)
         form = DigitalTakeupReportForm(data=self.request.GET.dict())
         if not form.is_valid():
             form = DigitalTakeupReportForm(data={})
             assert form.is_valid(), 'Empty form should be valid'
 
+        self.exclude_private_estate = form.cleaned_data['private_estate'] == 'exclude'
```

Comment:
Given `self.exclude_private_estate` is already set to true, I think it might be more readable to explicitly show that we are just changing this to False if the private estate value is not exclude:
```
if form.cleaned_data['private_estate'] != 'exclude':
    self.exclude_private_estate = False
```

Super minor thing but took me a second longer to read it in the current form.

Reply:
ah the reason why it's pre-set to true is so that the attribute always exists. so doing `self. exclude_private_estate` will not raise an error. static analysis would complain if an attribute is set on self for the first time outside of the __init__ method. happy to go with you suggestion tho.